src/config.json

```diff
@@ -663,6 +663,74 @@
 			}
 		}
 	],
+	"tutorial": [
+		{
+			"path": "/tutorial",
+			"name": {
+				"en": "Learn Preact"
+			}
+		},
```

Comment:
Extracting out like this a) makes it a bit more consistent with all the other content and b) ensures the tutorial pages are prerendered with proper titles. At the moment they're all `Preact - Preact Tutorial` insead of `Virtual DOM - Preact Tutorial`, etc.